crates/sui-protocol-config/src/lib.rs

```diff
@@ -1373,6 +1373,8 @@ impl ProtocolConfig {
                 }
                 23 => {
                     cfg.feature_flags.loaded_child_object_format_type = true;
+                    cfg.feature_flags.narwhal_new_leader_election_schedule = true;
+                    cfg.consensus_bad_nodes_stake_threshold = Some(20);
```

Comment:
Now that this is hitting mainnet, maybe worth adding a short comment why this value (in the range of 0-33) was chosen?